shared/chat/conversation/messages/wrapper/exploding-meta/index.js

```diff
@@ -115,10 +115,10 @@ class ExplodingMeta extends React.Component<Props, State> {
     let children
     switch (this.state.mode) {
       case 'countdown':
-        let bombIconSize = isMobile ? 22 : 16
+        let timerIconSize = isMobile ? 22 : 16
         if (isAndroid) {
           // icon is 24 high and clips edge of container on android. workaround
```

Comment:
Do you know if this is still needed?

Reply:
@chrisnojima Do you know if this is still needed?

Reply:
...taking a look.

Reply:
lets touch this stuff very little, i'm in the middle of all of this